### PR TITLE
xy_19

### LINKED ISSUE #297: xy_19
Description: Creating a single XY plot with three different sets of data that each have their own y-axis

NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/xy_19.ncl
NCL Plots: https://www.ncl.ucar.edu/Applications/Images/xy_19_1_lg.png https://www.ncl.ucar.edu/Applications/Images/xy_19_2_lg.png
https://www.ncl.ucar.edu/Applications/Images/xy_19_3_lg.png

Category: XY

### PR DESCRIPTION
**Pull request by hCraker**
Closes #297 
NCL Plots: https://www.ncl.ucar.edu/Applications/Images/xy_19_1_lg.png https://www.ncl.ucar.edu/Applications/Images/xy_19_2_lg.png
https://www.ncl.ucar.edu/Applications/Images/xy_19_3_lg.png

Python Plots:
image
image
image

### CODE REVIEW

File: Plots/XY/NCL_xy_19.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_19.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/xy_19_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/xy_19_2_lg.png and https://www.ncl.ucar.edu/Applications/Images/xy_19_3_lg.png
+
+"""
+
+###############################################################################
+# Import packages:
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
```
Comment (michaelavs) [inline Plots/XY/NCL_xy_19.py:18]:
```suggestion

```
Only change I found! Looks good overall!

**Review by michaelavs — APPROVED** [review]

**Review by anissa111 — COMMENTED** [review]
Sorry this review took so long, with the issues involving testing forked branches against master, this kept getting put off. This looks great! My only comment is a typo that I noted below. After that, this PR should be good to go!

**Review by anissa111 — APPROVED** [review]

**Comment by pilotchute** [comment]
I don't think that there will be a status reported for this pull request. I may be wrong, but I've been seeing no status reports from github actions on pull requests from forked repos.

**Comment by anissa111** [comment]
> I don't think that there will be a status reported for this pull request. I may be wrong, but I've been seeing no status reports from github actions on pull requests from forked repos.

Right, I'm not sure why that is. Possibly because before the only checks being run on forked branches were the ones that got removed last week? To check the forked PRs, I've been cloning the branch and letting the actions run (see test_xy_19_again) and activating the cloned branch on RTD.

**Pull request merged**